docs/supported-libraries.md

```diff
@@ -225,6 +225,15 @@ For this reason, the following instrumentations are disabled by default:
 - `jdbc-datasource` which creates spans whenever the `java.sql.DataSource#getConnection` method is called.
 - `dropwizard-metrics` which might create a very low quality metrics data, because of lack of label/attribute support
   in the Dropwizard metrics API.
+- `jaxws-jws-api`
+- `mybatis`
+- `spring-boot-actuator-autoconfigure`
```

Comment:
This enables micrometer metric bridging, see https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/9762